executor/distsql.go

```diff
@@ -309,6 +319,19 @@ func startSpanFollowsContext(ctx context.Context, operationName string) (opentra
 	return span, opentracing.ContextWithSpan(ctx, span)
 }
 
+func rebuildIndexRanges(ctx sessionctx.Context, is *plan.PhysicalIndexScan, idxCols []*expression.Column, colLens []int) (ranges []*ranger.Range, err error) {
```

Comment:
Why name is `rebuild`?

Reply:
Because the behavior is `set the ranges of executor again`. So i name it `rebuild`

Reply:
Please add comments for it.